### PR TITLE
Update "vbr" & "cbr" to "variable" & "constant" per TAG review.

### LINKED ISSUE #203: Use non-abbreviated terms in bitrates
In https://github.com/w3ctag/design-reviews/issues/540, we suggested that non-abbreviated forms for the bitrate enum would be better for user ergonomics; but it looks like that change has not been made yet.

Roughly this:
s/vbr/variable/
s/cbr/constant/

This issue is to track that suggestion.

### PR DESCRIPTION
**Pull request by sizeak**
A request was raised in the [W3C TAG review](https://github.com/w3ctag/design-reviews/issues/540) for https://github.com/w3c/mediacapture-record/pull/185 to change the `BitrateMode` enum values to "constant" and "variable".

***
<a href="https://pr-preview.s3.amazonaws.com/sizeak/mediacapture-record/pull/210.html" title="Last updated on Dec 16, 2020, 2:53 PM UTC (59f8565)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/mediacapture-record/210/081eda8...sizeak:59f8565.html" title="Last updated on Dec 16, 2020, 2:53 PM UTC (59f8565)">Diff</a>

### CODE REVIEW

**Review by dontcallmedom — APPROVED** [review]

**Comment by sizeak** [comment]
Hi, I don't have write access @dontcallmedom , would you mind merging it please? Thanks.

**Comment by dontcallmedom** [comment]
@simoncent can you confirm my impression that no browser has implemented this yet?

(if there were existing implementations shipping, I would want confirmation from implementors before merging)

**Comment by sizeak** [comment]
I have an open PR for Chrome/Chromium which I will update with this change, but I am unaware of any other implementations.

**Pull request merged**
